### ISSUE TITLE
Assets that have never been audited don't show in "Due for Audit" or "Overdue for Audit".

### ISSUE THREAD

**Issue by CMCLAuto** [grokability/snipe-it#9841]
I'm not sure if this is a bug report, a feature request or just intended function, but here goes.

#### Expected Behavior (or desired behavior if a feature request)

Assets that have never been audited would show up in the "Due for Audit" report.

-----

#### Actual Behavior

Assets with a blank Last Audit field can't be found with audit reports.

The first version of Snipe-IT that we used didn't have the Audit feature, so we've got hundreds of assets with a blank Last Audit field. As a result, they just never come due for an audit. New assets also have a blank Last Audit field (which makes sense, audits can't be assumed), but that means they don't show up as due for audit either. To get around that part we normally audit something as soon as it's labeled, but if one gets missed it just permanently drops of the radar.

-----

#### Please confirm you have done the following before posting your bug report:

- [Y] I have enabled debug mode 
- [Y] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

-----
#### Provide answers to these questions:

- Is this a fresh install or an upgrade? 
Upgrade
- Version of Snipe-IT you're running
v5.1.8 -build 6227
- Version of PHP you're running
v7.2.14
- Version of MySQL/MariaDB you're running
MariaDB 10.4.17
- What OS and web server you're running Snipe-IT on
Windows Server 2019, IIS 10
- What method you used to install Snipe-IT (install.sh, manual installation, docker, etc)
Git and upgrade.php
- WITH DEBUG TURNED ON, if you're getting an error in your browser, include that error
No error thrown
- What specific Snipe-IT page you're on, and what specific element you're interacting with to trigger the error
Due for Audit and Overdue for Audit
- If a stacktrace is provided in the error, include that too.
No error thrown
- Any errors that appear in your browser's error console.
No error thrown
- Confirm whether the error is reproducible on the demo: https://snipeitapp.com/demo.
Yes, actually. None of the assets in the demo have a "Last Audit" date. Running an audit report on the demo will come back blank.
- Include any additional information you can find in `storage/logs` and your webserver's logs.
No error thrown
- Include what you've done so far in the installation, and if you got any error messages along the way.
Standard installation, no error thrown
- Indicate whether or not you've manually edited any data directly in the database
Nothing edited.

Please do not post an issue without answering the related questions above. If you have opened a different issue and already answered these questions, answer them again, once for every ticket. It will be next to impossible for us to help you.

https://snipe-it.readme.io/docs/getting-help

I imagine this could be intentional? Maybe there are some assets that people don't want showing up in audit reports? All I know is we've got hundreds of assets that need to be audited and they don't show up in the report because a blank field isn't more then a year in the past.

**Comment by CMCLAuto** [comment]
We recently updated to 5.2.0 build - build 6339 (master) and noticed that newly created assets are now automatically getting a "Next Audit Date" for one year after creation. Very cool! Thank you, that stops any new assets from falling through the audit cracks from here on out. Was this added in 5.2, or was I just not noticing it under 5.1?

Still hoping for a way to find never-audited assets in a report in the future. Perhaps after 6.0 isn't consuming everyone's time?

**Comment by CMCLAuto** [comment]
Since current versions of Snipe automatically fill in the next_audit_date field, my problem of assets not showing up as due for audit is just a legacy issue now. For anyone who'd like their old assets to show up in the Due For Audit report, here's a SQL query I used to set my legacy assets up with a next audit date like new assets get. 

_Please back up your database before you run this!_  I'm not great with SQL and I certainly can't help anyone fix their database. 

Note that this query assumes the Snipe database name is _snipeit_ and sets the next audit date to November 16th, 2021(since that's the date I ran the query).

`UPDATE snipeit.assets SET next_audit_date=2021-11-16 WHERE next_audit_date IS NULL`

Hopefully that helps someone! Now if you'll excuse me, I suddenly have 811 more assets to go audit...
